erp/schema.py

```diff
@@ -549,7 +560,7 @@
         "is_a11y": True,
         "label": "Dispositif d'appel à l'entrée",
         "help_text": mark_safe(
-            "Existe-t-il un dispositif comme une sonnette pour permettre à quelqu'un ayant besoin de la rampe ou d'une aide humaine de signaler sa présence&nbsp;?"
+            "Existe-t-il un dispositif pour permettre à quelqu'un signaler sa présence&nbsp;?"
```

Comment:
I'd go for *signaler sa présence **à l'entrée***

Reply:
As you want :)